app/models/project.rb

```diff
@@ -7,3 +7,42 @@ class Project < ActiveRecord::Base
   has_many :rankings, through: :participants
   has_one :matching
 end
+
+
+public def formatted_preferences
+    preferences = []
+    self.rankings.each do |ranking|
+      person = ranking.participant.email
+      timeslot = ranking.project_time.date_time
+      timeslot_formatted = timeslot.strftime('%Y-%m-%d %H:%M')
+      if ranking.rank != 0
+        row = {"person_name": person, "timeslot": timeslot_formatted, "rank": ranking.rank}
+        preferences.push(row)
+      end
+    end
+
+    preferences
+end
+
+public def email_participants (subject, body)
```

Comment:
should call this `email_matching_to_participants` given there's another email method that does emailing of links to rank

Reply:
done